docs/getting_started.rst

```diff
@@ -53,7 +53,35 @@ This is a guideline however, and a ``Dataset`` seen in practice might include ma
 The worked examples below show how to access and visualize data like this using Xarray. They also demonstrate
 several of the sgkit conventions in place for representing common genetic quantities.
 
-.. ipython:: python
+.. jupyter-execute::
+    :hide-code:
+    :hide-output:
+
+    import os
+    import sys
+    raise ValueError(os.getcwd())
+    sys.path.insert(0, os.path.abspath(".."))
```

Comment:
Need to set the PYTHONPATH in conf.py: Figure out why sgkit isn't on PYTHONPATH in jupyter kernel

Reply:
Ah thanks!